lib/bolt/config/options.rb

```diff
@@ -239,11 +239,21 @@ module Options
               "name" => {
                 description: "The name of the module.",
                 type: String
+              },
+              "version_requirement" => {
+                description: "The version requirement for the module. Accepts either a specific version, version "\
```

Comment:
It might be helpful to have examples of those formats.

`Accepts a specific version (1.2.3), version shorthand (1.2.x), or a version range (>= 1.2.0)`